ansible/library/exabgp.py

```diff
@@ -82,6 +92,7 @@ def run_command():
     }
 '''
 
+# exabgp v3 configuration file format
```

Comment:
`exabgp v4` means IPv4. So, the `v3` here is misleading.

Reply:
I think the file here was referring to ExaBGP `v3` in the context of version 3. And `v4` is for ExaBGP [v4] (https://github.com/Exa-Networks/exabgp/releases/tag/4.0.1) which supported Python 3. I can probably modify the variable name to indicate `release v4` instead of just `v4`.

Reply:
Modified variable names to disambiguate.